mayastor/src/lvs/lvs_pool.rs

```diff
@@ -244,6 +257,85 @@ impl Lvs {
         }
     }
 
+    /// imports a pool based on its name, uuid and base bdev name
+    pub async fn import_from_args(args: PoolArgs) -> Result<Lvs, Error> {
+        if args.disks.len() != 1 {
+            return Err(Error::Invalid {
+                source: Errno::EINVAL,
+                msg: format!(
+                    "invalid number {} of devices {:?}",
+                    args.disks.len(),
+                    args.disks
+                ),
+            });
+        }
+
+        // default to aio if no specific driver scheme is specified
+        let disks = args
```

Comment:
I think we want to move this into the try_from() part as well.

Reply:
Similar I want to keep the PoolArgs generic for both lvm and lvs that way we don't repeat the common checks